pkg/util/resize/configspec.go

```diff
@@ -482,6 +483,46 @@ func compareNPIV(
 	}
 }
 
+// compareVirtualNuma compares the virtual NUMA configurations set in the config spec.
+func compareVirtualNuma(
+	ci vimtypes.VirtualMachineConfigInfo,
+	cs vimtypes.VirtualMachineConfigSpec,
+	outCS *vimtypes.VirtualMachineConfigSpec) {
+	if cs.VirtualNuma == nil {
+		return
+	}
+
+	if ci.NumaInfo == nil {
+		outCS.VirtualNuma = &vimtypes.VirtualMachineVirtualNuma{
+			CoresPerNumaNode:       cs.VirtualNuma.CoresPerNumaNode,
+			ExposeVnumaOnCpuHotadd: cs.VirtualNuma.ExposeVnumaOnCpuHotadd,
+		}
+
+		return
+	}
+
+	outCS.VirtualNuma = &vimtypes.VirtualMachineVirtualNuma{}
+	if ci.NumaInfo != nil {
```

Comment:
` ci.NumaInfo` was already checked for `nil` on L495 so we can simply this a little bit.

Reply:
Thanks. Yep.